core/services/aws/cognitoTrigger/lambdas/customEmailTemplate.ts

```diff
@@ -0,0 +1,24 @@
+import { CustomMessageTriggerEvent, Context, Callback } from 'aws-lambda'
+
+function customEmailTemplateLambda(
+  event: CustomMessageTriggerEvent,
+  _: Context,
+  callback: Callback<CustomMessageTriggerEvent>
+): void {
+  if (event.triggerSource === 'CustomMessage_SignUp') {
+    event.response.emailSubject = 'Welcome to Blood Connect!'
+    event.response.emailMessage = `Hello ${event.request.userAttributes.name},<br/><br/>
+                                      Welcome! Please verify your email using the following code: ${event.request.codeParameter}.<br/><br/>
+                                      Thanks!`
+  } else if (event.triggerSource === 'CustomMessage_ForgotPassword') {
+    event.response.emailSubject = 'Reset your password for Blood Connect'
+    event.response.emailMessage = `Hello ${event.request.userAttributes.name},<br/><br/>
```

Comment:
Do not add email message in js file. Add it other file and read from here.
In future I believe this will be HTML template.

Reply:
All messages are kept in a separate constant file.

You will find wit this [Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/113/commits/6d856e5322c15066df61036318febceea7de3529)